astropy/units/quantity.py

```diff
@@ -901,6 +901,55 @@ def to_value(self, unit=None, equivalencies=[]):
     --------
     to_value : Get the numerical value in a given unit.
     """)
+    
+    def to_string(quantity,unit='human'):
+        """
+        Return value in a human readable unit
+
+        Parameters
+        ----------
+        quantity:
+            Value to be converted to human readable unit.
+
+        unit:string
+            the choice of user to get value in string or 
+            to get value with appropriate SI prefix.
+        """
+        if not quantity.isscalar:
+            raise Exception("given quantity is not scalar")
+        from astropy import units as u
+        from astropy.units import get_current_unit_registry
+        from astropy.utils import console
+        from math import isclose
+        ureg = get_current_unit_registry()
+        units = []
+        if quantity.unit.is_equivalent(u.s) and quantity > 1 * u.s:
```

Comment:
if quantity is less than 1 second then it is dealt like any other unit.This is done so as to include conversion to ms,us etc